### ISSUE TITLE
Cannot connect error

### ISSUE THREAD

**Issue by dengke80622** [bimmerconnected/ha_custom_component#71]
- Tested with 20230608.1 from hacs

- Tested with OS integrated 

- Same issue.

The integration shows "Failed to connect"
Region: China

Tested with new installation with vmware image.
Home Assistant 2023.9.2
Supervisor 2023.08.3
Operating System 10.5

Log data:
```
Logger: bimmer_connected.api.authentication
Source: components/bmw_connected_drive/config_flow.py:44
First occurred: 1:19:01 PM (53 occurrences)
Last logged: 1:35:40 PM

MyBMWAPIError due to HTTPStatusError: True -
```

**Comment by rikroe** [comment]
The custom component is not up to date. Please remove it and use the integrated one. The login to china was fixed in 2023.9.

**Comment by dengke80622** [comment]
> The custom component is not up to date. Please remove it and use the integrated one. The login to china was fixed in 2023.9.

Just have a new installation in vmware, with version 2023.9.2, without hacs installed, same issue.
The mybmw app works fine.

**Comment by rikroe** [comment]
Ah, then you probably have an issue with the captcha (see https://github.com/bimmerconnected/bimmer_connected/pull/564).
